resotocore/resotocore/web/api.py

```diff
@@ -106,7 +106,7 @@ def section_of(request: Request) -> Optional[str]:
     return section
 
 
-AlwaysAllowed = {"/", "/metrics", "/api-doc.*", "/system/.*", "/ui.*", "/ca/cert", "/notebook.*"}
+AlwaysAllowed = {"/", "/metrics", "/api-doc.*", "/system/.*", "/ui.*", "/ca/cert", "/notebook.*", "/events"}
```

Comment:
Are we sure? Events contain sensitive information like account IDs. Those shouldn't be public.

Reply:
They are not - authentication is handled in http response handler.
Created 2 lists to make the intent more clear.